test/GCHP/intTestCreate.sh

```diff
@@ -31,173 +27,148 @@
+# c24 geosfp TransportTracers
+create_rundir "2\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
 
-#=============================================================================
-# Standard run directories
-#=============================================================================
+# c24 merra2 fullchem tagO3
+create_rundir "4\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
 
-dir="gchp_fullchem_standard_merra2_c24"
-create_rundir "1\n1\n1\n${root}\n${dir}\nn\n" ${root} ${dir} ${log}
-ln -s ${root}/gchp.env ${root}/${dir}/gchp.env
-cp ${testDir}/gchp.slurm.sh ${root}/${dir}/gchp.slurm.sh
-sed -i -e "s/AutoUpdate_Diagnostics=.*/AutoUpdate_Diagnostics=ON/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/CS_RES=.*/CS_RES=24/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/Diag_Monthly=\"1\".*/Diag_Monthly=\"0\"/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/Diag_Frequency=\".*/Diag_Frequency=\"010000\"/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/Diag_Duration=\".*/Diag_Duration=\"010000\"/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/TOTAL_CORES=.*/TOTAL_CORES=24/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/NUM_NODES=.*/NUM_NODES=1/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/NUM_CORES_PER_NODE=.*/NUM_CORES_PER_NODE=24/" ${root}/${dir}/setCommonRunSettings.sh
+# DEBUG: Exit after creating a couple of rundirs if $quick is "yes"
+if [[ "x${quick}" == "xyes" ]]; then
+    cd ${thisDir}
+    exit 0
+fi
 
-#=============================================================================
-# Benchmark run directories
-#=============================================================================
+# c24 merra2 fullchem_standard
+create_rundir "1\n1\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
 
-dir="gchp_fullchem_benchmark_merra2_c48"
-create_rundir "1\n2\n1\n${root}\n${dir}\nn\n" ${root} ${dir} ${log}
-ln -s ${root}/gchp.env ${root}/${dir}/gchp.env
-cp ${testDir}/gchp.slurm.sh ${root}/${dir}/gchp.slurm.sh
-sed -i -e "s/CS_RES=.*/CS_RES=48/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/AutoUpdate_Diagnostics=.*/AutoUpdate_Diagnostics=ON/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/Diag_Monthly=\"1\".*/Diag_Monthly=\"0\"/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/Diag_Frequency=\".*/Diag_Frequency=\"010000\"/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/Diag_Duration=\".*/Diag_Duration=\"010000\"/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/TOTAL_CORES=.*/TOTAL_CORES=24/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/NUM_NODES=.*/NUM_NODES=1/" ${root}/${dir}/setCommonRunSettings.sh
-sed -i -e "s/NUM_CORES_PER_NODE=.*/NUM_CORES_PER_NODE=24/" ${root}/${dir}/setCommonRunSettings.sh
+# c24 merra2 fullchem_benchmark
+create_rundir "1\n2\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
+
+# c24 merra2 fullchem_RRTMG
+create_rundir "1\n8\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
```

Comment:
Can we add a placeholder for the carbon simulation here too? I realize it will currently fail because of some 47L files that are still used, so we can keep it commented out for now.

Reply:
Now added in commit 9f68416bf630f7e303e376f1868a0bd7121bcf53